models/staging/stg_elementary_dbt_invocations.sql

```diff
@@ -29,16 +29,17 @@ with
             , project_name
         from {{ source('raw_dbt_monitoring', 'dbt_invocations') }}
     )
-    , utils_dateadd as (
+
+    , dbt_dateadd as (
         select distinct
             invocation_id
             , job_id
             , job_name
             , job_run_id
-            , {{ dbt_utils.dateadd('hour', -3, 'run_started_at') }} as invocation_started_at
-            , {{ dbt_utils.dateadd('hour', -3, 'run_completed_at') }} as invocation_completed_at
-            , {{ dbt_utils.dateadd('hour', -3, 'run_started_at') }} as invocation_date
-            , {{ dbt_utils.dateadd('hour', -3, 'generated_at') }} as invocation_generated_at
+            , {{ dbt.dateadd("hour", -3, 'run_started_at') }} as invocation_started_at
+            , {{ dbt.dateadd('hour', -3, 'run_completed_at') }} as invocation_completed_at
```

Comment:
why does only one line have double quotes and the others lines have single quotes?